packages/compass-collection/src/components/collection-header/collection-header.tsx

```diff
@@ -47,26 +55,49 @@ const collectionHeaderTitleDBStyles = css({
   },
 });
 
-const collectionHeaderTitleCollectionStyles = css({
+const collectionHeaderTitleDBDarkStyles = css({
+  color: uiColors.green.light2,
+  flexShrink: 2,
+  flexBasis: 'auto',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  cursor: 'pointer',
+  textDecoration: 'none',
+  '&:hover,&:focus': {
+    textDecoration: 'underline',
+  },
+});
```

Comment:
Seems like we are duplicating quite a lot of styles between light and dark here, would it make sense to have only properties that change defined separately?

Reply:
I was also considering this but was wondering if it reduces code readability with lots of styles combined with each other.

Reply:
While it does introduce more complexity at the rendering level I think I would agree that separating the theme styles from the main component styles that both share is probably worth doing, or it's at least my preference. I think having general css rules like `flexShrink` or `overflow` in both of the styles can fairly easily lead to one style being out of sync with the other if one is changed without the other.

Here's how I was doing it in `find-in-page`:
https://github.com/mongodb-js/compass/blob/3bfdea024eff1067e282681d729039fb1c6301b4/packages/compass-find-in-page/src/components/find-in-page-input.tsx#L138
It's definitely more things going on in the rendering, but I think `cx` might help improve readability concerns.